backend/tests/integration/records/serializers/json/test_data.py

```diff
@@ -400,3 +401,8 @@ def test_data_elasticsearch_schema_serialize_literature(inspire_app):
         response_data = orjson.loads(response.data)
         response_data_hit = response_data["hits"]["hits"][0]["metadata"]
         assert response_data_hit["literature"] == expected_literature
+
+
+def test_data_format_date():
+    data_detail_schema = DataDetailSchema()
+    assert data_detail_schema.get_date({"creation_date": "2022-01-01"}) == "Jan 1, 2022"
```

Comment:
Here it would be better if we can achieve the whole flow-> similar to how we do here https://github.com/inspirehep/inspirehep/blob/master/backend/tests/integration/records/serializers/json/test_data.py#L255, so as to simulate the whole flow of creating a record that gets serialized correctly when requesting /data/<recid>